src/main/java/piano/keyboard/keyboardaudio/Notes.java

```diff
@@ -4,12 +4,12 @@
 
 public interface Notes
 {
-	int C = 60;
+	int C =57;
```

Comment:
These values were wrong?

Reply:
I added three keys to the bottom of the piano that are before C so I updated the note numbers and shifted them by three